### PR TITLE
Support html element issue

### PR DESCRIPTION
**Pull request by dadaa**

### CODE REVIEW

File: extension/lib/webcompat.js [hunk 1]
```diff
@@ -81,6 +82,25 @@ class WebCompat {
...
+  /**
+   * @param {String} elementName -
+   *                e.g. div, main, aside
+   * @param {Array} browsers -
+   *                e.g. [{ id: "firefox", name: "Firefox", version: "68" }, ...]
+   * @return {Object} if no issue found, return null.
+   */
+  getHTMLElementIssue(elementName, browsers) {
+    const database = this._webCompatData.html.elements;
+    const summary = this._getCompatSummary(browsers, database, elementName);
```
Comment (captainbrosset) [inline extension/lib/webcompat.js:94]: We might need to do some filtering for custom elements. Go to youtube.com for a good example of why this is important.
They use a lot of custom elements like `ytd-button-renderer` and others.
Currently the extension does not handle this well and fires a warning for those. It should not.

I think you can do this to check: `window.customElements.get(element.localName)`, where `window` is the content page window object.

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: extension/experiments/inspectedNode/api.js [hunk 2]
```diff
@@ -83,6 +91,10 @@ this.inspectedNode = class extends ExtensionAPI {
     return {
       experiments: {
         inspectedNode: {
+          async getNodeName(clientId) {
+            return _getNodeName(clientId);
```
Comment (captainbrosset) [inline extension/experiments/inspectedNode/api.js]: Or you could fix the issue I reported before here. By not only returning the node name, but also a boolean that says if the element is custom or not.

**Review by captainbrosset — APPROVED** [review]

File: extension/lib/webcompat.js (hunk 1 above)
Comment (dadaa) [reply extension/lib/webcompat.js:94]: Thank you for catching this!
Yeah, indeed. Will address it.

File: extension/experiments/inspectedNode/api.js (hunk 2 above)
Comment (dadaa) [reply extension/experiments/inspectedNode/api.js]: Yeah, I like this solution rather, thanks!

**Review by captainbrosset — APPROVED** [review]

**Comment by dadaa** [comment]
Thank you very much!

**Pull request merged**